randovania/games/am2r/gui/ui_files/preset_am2r_energy.ui

```diff
@@ -0,0 +1,221 @@
+             <property name="minimum">
+              <number>2</number>
+             </property>
+             <property name="maximum">
+              <number>1000</number>
+             </property>
+             <property name="value">
+              <number>100</number>
+             </property>
+            </widget>
+           </item>
+           <item row="2" column="0">
+            <widget class="QLabel" name="energy_tank_capacity_label">
+             <property name="text">
+              <string>Energy per tank</string>
+             </property>
+            </widget>
+           </item>
+           <item row="0" column="0" colspan="2">
+            <widget class="QLabel" name="energy_tank_capacity_description">
+             <property name="text">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure how much energy each Energy Tank you collect will provide. Your base energy is always this quantity, minus 1.&lt;/p&gt;&lt;p&gt;While logic will respect this value, only the original value (100) has been tested.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+             </property>
+             <property name="wordWrap">
+              <bool>true</bool>
+             </property>
+            </widget>
+           </item>
+           <item row="1" column="0" colspan="2">
+            <spacer name="energy_tank_capacity_spacer">
+             <property name="orientation">
+              <enum>Qt::Vertical</enum>
+             </property>
+             <property name="sizeType">
+              <enum>QSizePolicy::Fixed</enum>
+             </property>
+             <property name="sizeHint" stdset="0">
+              <size>
+               <width>20</width>
+               <height>20</height>
+              </size>
+             </property>
+            </spacer>
+           </item>
+          </layout>
+         </widget>
+        </item>
+        <item>
+         <widget class="QGroupBox" name="suit_dr_box">
+          <property name="title">
+           <string>Suit Damage Reduction</string>
+          </property>
+          <layout class="QGridLayout" name="constant_environment_damage_layout">
+           <item row="5" column="0">
+            <widget class="QLabel" name="first_suit_label">
+             <property name="text">
+              <string>First Suit Damage Reduction</string>
+             </property>
+            </widget>
+           </item>
+           <item row="4" column="0" colspan="2">
+            <widget class="QLabel" name="suit_dr_description">
+             <property name="text">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure Damage Reduction (DR) for suit count. AM2R only cares about suit count not order.&lt;/p&gt;&lt;p&gt;0% you will take full damage (equal to suitless). 100% you will take no damage.&lt;/p&gt;&lt;p&gt;While logic will respect this value, only the vanilla values (50%, 75%) have been tested.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
The DR is now useless as you dont mention it anywhere, also wording sounded slightly off.
```suggestion
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure Damage Reduction for suit count. AM2R only cares about suit count not order.&lt;/p&gt;&lt;p&gt;0% will make you take full damage (equal to suitless). 100% will make you take no damage.&lt;/p&gt;&lt;p&gt;While logic will respect this value, only the vanilla values (50%, 75%) have been tested.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```